test/README.md

```diff
@@ -1 +1 @@
-These tests are taken from repository https://github.com/datastax/nodejs-driver/tree/master
+Tests in ``./integration``, ``./other`` and ``./unit-not-supported`` are taken from repository https://github.com/datastax/nodejs-driver/tree/master
```

Comment:
Are those double backticks on purpose? Why?

Reply:
that's MD file, so marks given folders as code in markdown. Similar approach as in https://github.com/scylladb-zpp-2024-javascript-driver/scylladb-javascript-driver/blob/a9165c6a3d0b663bb0b2587f1d8fe81ea5a86b94/examples/README.md?plain=1#L1-L5